### PR TITLE
[COMDLG32] pt-PT: Remove accel key from Ok buttons.

### PR DESCRIPTION
**Pull request by zecarlos1957**
## Purpose

Remove accel key from Ok buttons.
Fix accel collisions

### CODE REVIEW

File: dll/win32/comdlg32/lang/cdlg_Pt.rc [hunk 1]
```diff
@@ -288,7 +288,7 @@ FONT 8, "MS Shell Dlg"
     CONTROL "", IDC_COLOR_GRAPH, "STATIC", WS_BORDER | SS_SIMPLE | WS_TABSTOP | WS_GROUP, 152, 4, 118, 116
     CONTROL "", IDC_COLOR_LUMBAR, "STATIC", SS_SIMPLE | WS_TABSTOP | WS_GROUP, 278, 4, 8, 116
     CONTROL "", IDC_COLOR_RESULT, "STATIC", SS_SIMPLE | WS_TABSTOP | WS_GROUP, 152, 124, 40, 26
-    DEFPUSHBUTTON "&OK", IDOK, 4, 167, 50, 14, BS_DEFPUSHBUTTON | WS_GROUP | WS_TABSTOP //FIXME:accelerator collision &O
+    DEFPUSHBUTTON "OK", IDOK, 4, 167, 50, 14, BS_DEFPUSHBUTTON | WS_GROUP | WS_TABSTOP
```
Comment (JoachimHenze) [inline dll/win32/comdlg32/lang/cdlg_Pt.rc:291]: correct for the pick-color-dlg. It doesn't have an accel on the ok button in XPSP3.

File: dll/win32/comdlg32/lang/cdlg_Pt.rc [hunk 2]
```diff
@@ -246,7 +246,7 @@ FONT 8, "MS Shell Dlg"
     LTEXT           "&Tamanho:", stc3, 179, 3, 32, 9
     COMBOBOX        cmb3, 179, 13, 32, 76, CBS_OWNERDRAWFIXED | CBS_HASSTRINGS | CBS_DISABLENOSCROLL |
                                            WS_VSCROLL | WS_TABSTOP | CBS_SIMPLE | CBS_SORT
-    DEFPUSHBUTTON   "&OK", IDOK, 218, 6, 50, 14, WS_GROUP | WS_TABSTOP | BS_DEFPUSHBUTTON
+    DEFPUSHBUTTON   "OK", IDOK, 218, 6, 50, 14, WS_GROUP | WS_TABSTOP | BS_DEFPUSHBUTTON
```
Comment (JoachimHenze) [inline dll/win32/comdlg32/lang/cdlg_Pt.rc:249]: correct for the coose-font-dlg, it doesn't have an accelerator here on XPSP3:
image

File: dll/win32/comdlg32/lang/cdlg_Pt.rc [hunk 3]
```diff
@@ -394,8 +394,8 @@ BEGIN
     COMBOBOX cmb3, 52, 126, 112, 112, CBS_DROPDOWNLIST | CBS_SORT | WS_VSCROLL | WS_GROUP | WS_TABSTOP
     GROUPBOX "Orientação", grp1, 180, 92, 100, 56, WS_GROUP
     ICON "", ico1, 195, 112, 18, 20, WS_GROUP
-    AUTORADIOBUTTON "&Retrato", rad1, 224, 106, 52, 12, WS_GROUP | WS_TABSTOP
-    AUTORADIOBUTTON "&Paisagem", rad2, 224, 126, 52, 12//FIXME: accelerator collision &P
+    CONTROL "&Retrato", rad1,"Button", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 224, 106, 52, 12
+    CONTROL "Pai&sagem", rad2,"Button", BS_AUTORADIOBUTTON, 224, 126, 52, 12
```
Comment (JoachimHenze) [inline dll/win32/comdlg32/lang/cdlg_Pt.rc:398]: This change-request is mandatory! You created a merge conflict here!
```suggestion
    AUTORADIOBUTTON "&Retrato", rad1, 224, 106, 52, 12, WS_GROUP | WS_TABSTOP
    AUTORADIOBUTTON "Pai&sagem", rad2, 224, 126, 52, 12
```

**Review by JoachimHenze — CHANGES_REQUESTED** [review]
see the mandatory change-request!

File: dll/win32/comdlg32/lang/cdlg_Pt.rc [hunk 4]
```diff
@@ -301,30 +301,30 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
...
+    LTEXT "&Procurar:", -1, 4, 8, 52, 8
     EDITTEXT edt1, 57, 7, 148, 12, ES_AUTOHSCROLL | WS_BORDER | WS_GROUP | WS_TABSTOP
     CHECKBOX "Palavra &inteira", chx1, 4, 26, 140, 12, BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP
     CHECKBOX "&Maiúsculas/minúsculas", chx2, 4, 42, 140, 12, BS_AUTOCHECKBOX | WS_TABSTOP
     GROUPBOX "Direcção", grp1, 147, 21, 58, 38
-    AUTORADIOBUTTON "&Acima", rad1, 151, 30, 48, 12, WS_CHILD | WS_VISIBLE | WS_GROUP | WS_TABSTOP//FIXME: accelerator collision &A
-    AUTORADIOBUTTON "A&baixo", rad2, 151, 44, 48, 12, WS_CHILD | WS_VISIBLE | WS_TABSTOP
-    DEFPUSHBUTTON "&Procurar", IDOK, 212,  6, 60, 14, WS_GROUP | WS_TABSTOP | BS_DEFPUSHBUTTON //FIXME: accelerator collision &P
+    CONTROL "&Acima", rad1, "BUTTON", BS_AUTORADIOBUTTON | WS_CHILD | WS_VISIBLE | WS_GROUP | WS_TABSTOP, 151, 30, 48, 12
+    CONTROL "A&baixo", rad2, "BUTTON", BS_AUTORADIOBUTTON | WS_CHILD | WS_VISIBLE | WS_TABSTOP, 151, 44, 48, 12
```
Comment (JoachimHenze) [inline dll/win32/comdlg32/lang/cdlg_Pt.rc:310]: mandatory!
```suggestion
    AUTORADIOBUTTON "&Acima", rad1, 151, 30, 48, 12, WS_CHILD | WS_VISIBLE | WS_GROUP | WS_TABSTOP
    AUTORADIOBUTTON "A&baixo", rad2, 151, 44, 48, 12, WS_CHILD | WS_VISIBLE | WS_TABSTOP
```

File: dll/win32/comdlg32/lang/cdlg_Pt.rc (hunk 1 above)
Comment (HBelusca) [reply dll/win32/comdlg32/lang/cdlg_Pt.rc:291]: Yep usually OK and Cancel won't have these. (Since this job is done by the ENTER and ESC keys already.)

**Review by JoachimHenze — APPROVED** [review]
Thanks for applying the recommended changes. Approved now. Ftr: **I did not double-check the correctness for the accel-stripping in the PRINT_SETUP-dlg, because that one would require me to install a printer on my OS, which I don't have in mind.** @HBelusca there are some exceptions to the general-rule of OK/CANCEL, e.g. in some of the MS file-open/file-close-dialogs, that's why I do often double-check such changes.

File: dll/win32/comdlg32/lang/cdlg_Pt.rc [hunk 5]
```diff
@@ -156,12 +156,12 @@ FONT 8, "MS Shell Dlg"
...
     COMBOBOX cmb1, 6, 114, 90, 60, CBS_DROPDOWNLIST | CBS_AUTOHSCROLL | WS_BORDER | WS_VSCROLL | WS_TABSTOP
-    LTEXT "Con&troladores:", stc4, 110, 104, 92, 9
+    LTEXT "&Controladores:", stc4, 110, 104, 92, 9
     COMBOBOX cmb2, 110, 114, 92, 68, CBS_DROPDOWNLIST | CBS_OWNERDRAWFIXED | CBS_AUTOHSCROLL | CBS_SORT | CBS_HASSTRINGS | WS_BORDER | WS_VSCROLL | WS_TABSTOP
     DEFPUSHBUTTON "Abrir", IDOK, 208, 6, 60, 14, BS_DEFPUSHBUTTON | WS_GROUP | WS_TABSTOP
     PUSHBUTTON "Cancelar", IDCANCEL, 208, 24, 60, 14, WS_GROUP | WS_TABSTOP
-    PUSHBUTTON "&Ajuda", pshHelp, 208, 46, 60, 14, WS_GROUP | WS_TABSTOP //FIXME: accelerator collision &A, I guess this should be &j
-    CHECKBOX "&Apenas de leitura", chx1, 208, 68, 65, 12, BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP  //FIXME: accelerator collision &A
+    PUSHBUTTON "A&juda", pshHelp, 208, 46, 60, 14, WS_GROUP | WS_TABSTOP
+    CHECKBOX "&Apenas de leitura", chx1, 208, 68, 65, 12, BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP
```
Comment (HBelusca) [inline dll/win32/comdlg32/lang/cdlg_Pt.rc:164]:
```suggestion
    AUTOCHECKBOX "&Apenas de leitura", chx1, 208, 68, 65, 12, WS_GROUP
```
https://learn.microsoft.com/en-us/windows/win32/menurc/autocheckbox-control

File: dll/win32/comdlg32/lang/cdlg_Pt.rc [hunk 6]
```diff
@@ -198,7 +198,7 @@ FONT 8, "MS Shell Dlg"
     RADIOBUTTON "&Páginas", rad3, 16, 75, 60, 12
     DEFPUSHBUTTON "&Imprimir", IDOK, 206, 6, 56, 14, BS_DEFPUSHBUTTON | WS_GROUP | WS_TABSTOP
     PUSHBUTTON "Cancelar", IDCANCEL, 206, 24, 56, 14, WS_GROUP | WS_TABSTOP
-    PUSHBUTTON "Con&figurar", psh1, 206, 46, 56, 14, WS_GROUP | WS_TABSTOP
+    PUSHBUTTON "&Configurar", psh1, 206, 46, 56, 14, WS_GROUP | WS_TABSTOP
```
Comment (HBelusca) [inline dll/win32/comdlg32/lang/cdlg_Pt.rc:201]: The `WS_GROUP | WS_TABSTOP` may be redundant here... (to be double-checked)

File: dll/win32/comdlg32/lang/cdlg_Pt.rc [hunk 7]
```diff
@@ -217,9 +217,9 @@ FONT 8, "MS Shell Dlg"
     LTEXT "[none]", stc1, 35, 30, 120, 9
     RADIOBUTTON "Impressora &específica", rad2, 16, 44, 150, 12
     COMBOBOX cmb1, 35, 58, 145, 68, CBS_DROPDOWNLIST | CBS_OWNERDRAWFIXED | CBS_AUTOHSCROLL | CBS_SORT | CBS_HASSTRINGS | WS_BORDER | WS_VSCROLL | WS_TABSTOP
-    DEFPUSHBUTTON "&OK", IDOK, 206, 6, 56, 14, BS_DEFPUSHBUTTON | WS_GROUP | WS_TABSTOP
+    DEFPUSHBUTTON "OK", IDOK, 206, 6, 56, 14, BS_DEFPUSHBUTTON | WS_GROUP | WS_TABSTOP
```
Comment (HBelusca) [inline dll/win32/comdlg32/lang/cdlg_Pt.rc:220]:
```suggestion
    DEFPUSHBUTTON "OK", IDOK, 206, 6, 56, 14, WS_GROUP | WS_TABSTOP
```
The `WS_GROUP | WS_TABSTOP` may be also redundant (to be double-checked)

File: dll/win32/comdlg32/lang/cdlg_Pt.rc [hunk 8]
```diff
@@ -198,7 +198,7 @@ FONT 8, "MS Shell Dlg"
     RADIOBUTTON "&Páginas", rad3, 16, 75, 60, 12
     DEFPUSHBUTTON "&Imprimir", IDOK, 206, 6, 56, 14, BS_DEFPUSHBUTTON | WS_GROUP | WS_TABSTOP
```
Comment (HBelusca) [inline dll/win32/comdlg32/lang/cdlg_Pt.rc:199]: Note: the BS_xxx is redundant

File: dll/win32/comdlg32/lang/cdlg_Pt.rc (hunk 2 above)
Comment (HBelusca) [inline dll/win32/comdlg32/lang/cdlg_Pt.rc:249]:
```suggestion
    DEFPUSHBUTTON   "OK", IDOK, 218, 6, 50, 14, WS_GROUP | WS_TABSTOP
```

File: dll/win32/comdlg32/lang/cdlg_Pt.rc (hunk 1 above)
Comment (HBelusca) [inline dll/win32/comdlg32/lang/cdlg_Pt.rc:291]:
```suggestion
    DEFPUSHBUTTON "OK", IDOK, 4, 167, 50, 14, WS_GROUP | WS_TABSTOP
```

File: dll/win32/comdlg32/lang/cdlg_Pt.rc [hunk 9]
```diff
@@ -301,34 +301,34 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
...
+    LTEXT "&Procurar:", -1, 4, 8, 52, 8
     EDITTEXT edt1, 57, 7, 148, 12, ES_AUTOHSCROLL | WS_BORDER | WS_GROUP | WS_TABSTOP
     CHECKBOX "Palavra &inteira", chx1, 4, 26, 140, 12, BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP
     CHECKBOX "&Maiúsculas/minúsculas", chx2, 4, 42, 140, 12, BS_AUTOCHECKBOX | WS_TABSTOP
     GROUPBOX "Direcção", grp1, 147, 21, 58, 38
-    AUTORADIOBUTTON "&Acima", rad1, 151, 30, 48, 12, WS_CHILD | WS_VISIBLE | WS_GROUP | WS_TABSTOP//FIXME: accelerator collision &A
+    AUTORADIOBUTTON "&Acima", rad1, 151, 30, 48, 12, WS_CHILD | WS_VISIBLE | WS_GROUP | WS_TABSTOP
     AUTORADIOBUTTON "A&baixo", rad2, 151, 44, 48, 12, WS_CHILD | WS_VISIBLE | WS_TABSTOP
-    DEFPUSHBUTTON "&Procurar", IDOK, 212,  6, 60, 14, WS_GROUP | WS_TABSTOP | BS_DEFPUSHBUTTON //FIXME: accelerator collision &P
+    DEFPUSHBUTTON "Pr&ocurar", IDOK, 212,  6, 60, 14, WS_GROUP | WS_TABSTOP | BS_DEFPUSHBUTTON
```
Comment (HBelusca) [inline dll/win32/comdlg32/lang/cdlg_Pt.rc:311]:
```suggestion
    DEFPUSHBUTTON "Pr&ocurar", IDOK, 212,  6, 60, 14, WS_GROUP | WS_TABSTOP
```

File: dll/win32/comdlg32/lang/cdlg_Pt.rc [hunk 10]
```diff
@@ -301,34 +301,34 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
...
 {
-    LTEXT "&Procurar:", -1, 4, 8, 52, 8 //FIXME: accelerator collision &P
+    LTEXT "&Procurar:", -1, 4, 8, 52, 8
     EDITTEXT edt1, 57, 7, 148, 12, ES_AUTOHSCROLL | WS_BORDER | WS_GROUP | WS_TABSTOP
     LTEXT "Substituir po&r:", -1, 4, 26, 52, 8
     EDITTEXT edt2, 57, 24, 148, 12, ES_AUTOHSCROLL | WS_BORDER | WS_GROUP | WS_TABSTOP
     CHECKBOX "Palavra &inteira", chx1, 5, 46, 120, 12, BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP
     CHECKBOX "&Maiúsculas/minúsculas", chx2, 5, 62, 120, 12, BS_AUTOCHECKBOX | WS_TABSTOP
-    DEFPUSHBUTTON "&Procurar", IDOK, 212,  6, 60, 14, WS_GROUP | WS_TABSTOP | BS_DEFPUSHBUTTON //FIXME: accelerator collision &P
+    DEFPUSHBUTTON "Pr&ocurar", IDOK, 212,  6, 60, 14, WS_GROUP | WS_TABSTOP | BS_DEFPUSHBUTTON
```
Comment (HBelusca) [inline dll/win32/comdlg32/lang/cdlg_Pt.rc:327]:
```suggestion
    DEFPUSHBUTTON "Pr&ocurar", IDOK, 212,  6, 60, 14, WS_GROUP | WS_TABSTOP
```

File: dll/win32/comdlg32/lang/cdlg_Pt.rc [hunk 11]
```diff
@@ -355,16 +355,16 @@ FONT 8, "MS Shell Dlg"
     GROUPBOX "Cópias", grp2, 160, 92, 120, 64, WS_GROUP
     LTEXT "Número de &cópias:", stc5, 168, 105, 68, 8
     ICON "", ico3, 210, 131, 76, 24, WS_GROUP
-    CONTROL "&Agrupar", chx2, "Button", BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP, 168, 118, 100, 12 //FIXME: accelerator collision &A
+    CONTROL "A&grupar", chx2, "Button", BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP, 168, 118, 100, 12
```
Comment (HBelusca) [inline dll/win32/comdlg32/lang/cdlg_Pt.rc:358]: Could be converted into `AUTOCHECKBOX ....` (see other existing examples above)
(And some other instances below.)

File: dll/win32/comdlg32/lang/cdlg_Pt.rc [hunk 12]
```diff
@@ -301,34 +301,34 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
...
 FONT 8, "MS Shell Dlg"
 {
-    LTEXT "&Procurar:", -1, 4, 8, 52, 8 //FIXME: accelerator collision &P
+    LTEXT "&Procurar:", -1, 4, 8, 52, 8
     EDITTEXT edt1, 57, 7, 148, 12, ES_AUTOHSCROLL | WS_BORDER | WS_GROUP | WS_TABSTOP
     CHECKBOX "Palavra &inteira", chx1, 4, 26, 140, 12, BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP
     CHECKBOX "&Maiúsculas/minúsculas", chx2, 4, 42, 140, 12, BS_AUTOCHECKBOX | WS_TABSTOP
     GROUPBOX "Direcção", grp1, 147, 21, 58, 38
-    AUTORADIOBUTTON "&Acima", rad1, 151, 30, 48, 12, WS_CHILD | WS_VISIBLE | WS_GROUP | WS_TABSTOP//FIXME: accelerator collision &A
+    AUTORADIOBUTTON "&Acima", rad1, 151, 30, 48, 12, WS_CHILD | WS_VISIBLE | WS_GROUP | WS_TABSTOP
```
Comment (HBelusca) [inline dll/win32/comdlg32/lang/cdlg_Pt.rc:309]: The `WS_VISIBLE` is redundant.

**Review by HBelusca — APPROVED** [review]
PR as is, is OK (stuff was already compiling fine I guess), but I also see stuff that could be simplified further. Maybe in separate commits or PRs since they don't pertain to accelerators removal? (@JoachimHenze weren't you trying to work on these already?)

File: dll/win32/comdlg32/lang/cdlg_Pt.rc (hunk 10 above)
Comment (JoachimHenze) [reply dll/win32/comdlg32/lang/cdlg_Pt.rc:327]: Yes, but I guess it is better to use a fresh PR for that, that would cover all languages, and not just pt-PT.

Comment (zecarlos1957) [reply dll/win32/comdlg32/lang/cdlg_Pt.rc:327]: Ok i can doit. Thanks.

File: dll/win32/comdlg32/lang/cdlg_Pt.rc (hunk 12 above)
Comment (zecarlos1957) [reply dll/win32/comdlg32/lang/cdlg_Pt.rc:309]: I doit in a new PR after this one merged.

**Pull request merged**